CMakeLists.txt

```diff
@@ -121,7 +121,7 @@ include(IrodsRunpathDefaults)
 
 set(IRODS_VERSION_MAJOR "5")
 set(IRODS_VERSION_MINOR "0")
-set(IRODS_VERSION_PATCH "1")
+set(IRODS_VERSION_PATCH "2")
```

Comment:
Since there are deprecations within the public API, we're going to need to bump the MINOR number instead of the PATCH number.

See https://semver.org/#spec-item-7 and https://semver.org/#how-should-i-handle-deprecating-functionality.